pom.xml

```diff
@@ -0,0 +1,163 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>uk.gov.companieshouse</groupId>
+	<artifactId>company-accounts-library</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>company-accounts-library</name>
+
+	<properties>
+
+		<!-- Encoding -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- Dependency Versions -->
+		<avro.version>1.8.1</avro.version>
+		<commons-codec-version>1.9</commons-codec-version>
+		<commons-io.version>2.4</commons-io.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<junit.version>4.12</junit.version>
```

Comment:
Should this be a newer version (JUnit 5? See Jupiter dependencies in web/api repositories).

Reply:
f19f8ed